bylaws/7-liability-indemnification-dissolution.md

```diff
@@ -0,0 +1,48 @@
+---
+status: draft
+weight: 7
+---
+# Liability, Indemnification, and Dissolution
+
+## Liability
+
+[CF Council][cfc] (CFC) Councilors and officers are not personally liable for actions undertaken in their official capacity, except in cases of willful misconduct or gross negligence.
+
+## Insurance
+
+**Decision-making:** The CFC is responsible for evaluating and deciding on the insurance needs of the Foundation. This includes assessing the necessity and feasibility of various insurance coverages.
+
+**Types of Insurance:** The CFC shall consider maintaining insurance policies appropriate for the Foundation’s activities and risks, such as Directors and Officers (D&O) Liability insurance, property insurance, and liability insurance for events.
+
+**Regular Review:** The insurance policies will be reviewed regularly, at least annually, to ensure they remain adequate and align with the Foundation's evolving needs and circumstances.
+
+## Indemnification
+
+The CF commits to protecting Councilors, officers, and other key personnel against legal risks associated with their roles. In cases arising from their duties, we provide indemnification for expenses and liabilities, under the following conditions:
+
+- Protection excludes acts of willful misconduct or gross negligence.
+- Those seeking indemnification must notify the Foundation promptly, enabling review and confirmation that the case aligns with our policy guidelines.
+
+## Dissolution and Asset Distribution
+
+In the unfortunate event that the Commonhaus Foundation (CF) must dissolve, we commit to a responsible and lawful closure process, prioritizing the continuity and integrity of our open-source projects.
+Given that CF projects represent collaborative effort of individual contributors, the dissolution process will respect the unique structure and community-driven nature of each project.
+
+## Dissolution Steps
+
+1. **Settling Liabilities:** All outstanding debts or obligations incurred by the Foundation will be paid off.
+2. **Prioritizing Project Continuity:** Before distributing any remaining assets, we will first seek to ensure the continuity of CF projects.
```

Comment:
I may be wrong, but I believe section 49.4 of the [Eclipse Foundation bylaws](https://www.eclipse.org/org/documents/eclipse-foundation-be-bylaws-en.pdf) has the same intent. **Net** assets must be allocated to another not-profit organization.

Reply:
Hi @nmcl , awesome to see you watching here :)

For reference, I talked about this issue with @ebullient and I'm satisfied that the exact wording and intent will be checked by someone more knowledgeable than me (hopefully a lawyer working in interest of our projects).

@kenfinnigan :

> I take that to mean projects can be "sold off" to pay debts if needed

That's precisely what I'm referring to. I understand that this is "fairly standard" and possibly unavoidable (I'd imagine laws would require adherence to such a principle), but I'm concerned that a mistake (even with best intentions) by an individual project could force a dissolution and therefore affect other projects; to ensure protection against such cases I suspect all projects would need to adhere to more strict processes - which CF claims to want to avoid.

So, in lack of clarity here I see a risk of undermining some core principles of CF.

For more context I made some examples on https://github.com/commonhaus/foundation-draft/discussions/68#discussioncomment-8413369

Again: I'm satisfied we can move on for now as @ebullient assured me we're aligned (and admittedly I'm not at all an expert in such matters) - just let's not forget working on this in due time.

Perhaps no further changes are necessary, but at least confirmation from an expert that this would indeed play out as we intend it.

Reply:
One example of a foundation dissolving: https://www.silverliningsinfo.com/automation/what-heck-happened-onf